bin/migrate

```diff
@@ -13,7 +15,7 @@ var db = mysql.createConnection({
 console.log("Running this script will initalize your database for pulldasher.");
 console.log("It will not drop any existing tables.");
 
-fs.readFile('./schema.sql', function(err, data) {
+fs.readFile('migrations/schema.sql', function(err, data) {
```

Comment:
I thought so originally, but the relative path comes from where the script is executed, which is the root directory. Which is kind of confusing because the `require('../config')` is relative to where the actual migrate script is located, but the fs.readFile path isn't.